docs/src/main/asciidoc/telemetry-opentracing-to-otel-tutorial.adoc

```diff
@@ -0,0 +1,504 @@
+
+This command generates the Maven project and imports the `smallrye-opentracing` extension, which
+includes the OpenTracing support and the default https://www.jaegertracing.io/[Jaeger] tracer.
+
+=== The application
+
+The Quarkus project has a single endpoint and the related class will look like this:
+
+[source,java]
+----
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return "Hello from RESTEasy Reactive";
+    }
+}
+----
+
+There is no OpenTracing specific code in the generated project, but the `smallrye-opentracing` extension is present and enabled by default, and it will automatically instrument the code.
+
+Let's start the Jaeger-all-in-one Docker image, where we will retrieve and see the captured traces:
+
+[source,bash]
+----
+docker run -e COLLECTOR_OTLP_ENABLED=true -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 4317:4317 -p 4318:4318 -p 14250:14250 -p 14268:14268 -p 14269:14269 -p 9411:9411 jaegertracing/all-in-one:latest
+----
+
+At this point you can run the application with Quarkus dev mode:
+
+include::{includes}/devtools/dev.adoc[]
+
+If you call the http://localhost:8080/hello[`/hello` endpoint] the related traces can be retrieved in the Jaeger UI at this address: http://localhost:16686/
+
+They will look like this:
+
+image::ot-to-otel-1.png[alt=OpenTracing span,role="center"]
+
+== Big bang change from OpenTracing to OpenTelemetry
+
+This is the happiest path, in this case there is no manual instrumentation. We can do a big bang change from OpenTracing to OpenTelemetry without side effects.
+
+=== Change dependencies
+
+To migrate between the two frameworks, you must drop the old `quarkus-smallrye-opentracing` extension and replace it by the new the `quarkus-opentelemetry` extension in the pom.xml file:
+
+The legacy extension is removed from the project:
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-smallrye-opentracing</artifactId>
+</dependency>
+----
+
+The new one is added:
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-opentelemetry</artifactId>
+</dependency>
+----
+
+=== Application properties
+
+You should remove the old OpenTracing properties, starting with `quarkus.jaeger.*` from the `application.properties` file, like in this example:
+[source,application.properties]
+----
+#Legacy OpenTracing properties to be removed
+quarkus.jaeger.service-name=legume
+quarkus.jaeger.sampler-type=const
+quarkus.jaeger.sampler-param=1
+quarkus.jaeger.endpoint=http://localhost:14268/api/traces
+quarkus.jaeger.log-trace-context=true
+----
+If you use the default values in the OpenTelemetry properties, there is no necessity to include anything in the `application.properties` file.
```

Comment:
Add a new line here after the `----`. You never know how the other publishing tool will react.

Reply:
ok